artiq/coredevice/phaser.py

```diff
@@ -408,6 +408,45 @@ def set_leds(self, leds):
         """
         self.write8(PHASER_ADDR_LED, leds)
 
+    @kernel
+    def get_leds(self) -> TInt32:
+        """Get the state of the front panel LEDs.
+
+        :return: LED settings (6 bit)
+        """
+        state = self.read8(PHASER_ADDR_LED)
+        delay(20*us)  # slack
```

Comment:
Looks arbitrary and other drivers e.g. TTL, Sampler do not integrate such slack.